data/test/news/articles/c0000000003o.json

```diff
@@ -88,39 +85,29 @@
             "locator": "urn:bbc:pips:pid:p067gq6h",
             "blocks": [
               {
-                "type": "rawVideo",
-                "model": {
-                  "guidance": null,
-                  "isLive": false,
-                  "duration": "PT1M34S",
-                  "allowOffsiteEmbedding": false,
-                  "locator": "urn:bbc:pips:pid:p067gq6h"
-                }
-              },
-              {
-                "type": "altText",
-                "model": {
-                  "blocks": [
+                "type" : "aresMedia",
+                "model" : {
+                  "id" : "p067gq6h",
+                  "subType" : "clip",
+                  "format" : "audio_video",
+                  "title" : "Mark and Isla Royal Wedding",
+                  "synopses" : {
+                    "short" : "Stop the press! It’s a Mark and Isla Royal Wedding special…",
+                    "medium" : null,
+                    "long" : null
+                  },
+                  "imageUrl" : "ichef.bbci.co.uk/images/ic/$recipe/p067grwg.jpg",
+                  "embedding" : true,
+                  "warnings" : { },
+                  "advertising" : false,
+                  "versions" : [
                     {
-                      "type": "text",
-                      "model": {
-                        "blocks": [
-                          {
-                            "type": "paragraph",
-                            "model": {
-                              "text": "Vestibulum sagittis neque at consequat porttitor. Maecenas vitae arcu vitae augue lobortis porta eget non erat.",
-                              "blocks": [
-                                {
-                                  "type": "fragment",
-                                  "model": {
-                                    "text": "Vestibulum sagittis neque at consequat porttitor. Maecenas vitae arcu vitae augue lobortis porta eget non erat.",
-                                    "attributes": []
-                                  }
-                                }
-                              ]
-                            }
-                          }
-                        ]
+                      "versionId" : "p067gq6k",
+                      "types" : [ "Original" ],
+                      "duration" : 94,
```

Comment:
Is duration always going to be in seconds?

Reply:
Yes